### PR TITLE
Pluralize correctly

### PR DESCRIPTION
**Pull request by joshualai9922**
- Fixed plural issues for report
 - Start Time shown on report will be always based on viewer's timezone. Changed to 24hr format and timezone will be displayed.
 - Retry mechanism for verbose mode

- [x] I've kept this PR as small as possible (~500 lines) by splitting it into PRs with manageable chunks of code
- [x] I've requested reviews from 1 reviewer
- [x] I've tested existing features (website scan, sitemap, custom flow) in both node index and cli
- [x] I've synced this fork with GovTechSG repo
- [ ] I've added/updated unit tests
- [ ] I've added/updated any necessary dependencies in `package[-lock].json` `npm audit`, portable installation on GitHub Actions

### CODE REVIEW

**Comment by younglim** [comment]
Open a new PR without Retry mechanism for verbose mode

File: constants/common.js
Comment (joshualai9922) [inline constants/common.js:1]: KIV: Verbose retry mechanism from point of failure

File: constants/constants.js
Comment (joshualai9922) [inline constants/constants.js:1]: KIV: Verbose retry mechanism from point of failure

File: crawlers/crawlSitemap.js
Comment (joshualai9922) [inline crawlers/crawlSitemap.js:1]: KIV: Verbose retry mechanism from point of failure

File: crawlers/crawlIntelligentSitemap.js
Comment (joshualai9922) [inline crawlers/crawlIntelligentSitemap.js:1]: KIV: Verbose retry mechanism from point of failure

File: crawlers/pdfScanFunc.js
Comment (joshualai9922) [inline crawlers/pdfScanFunc.js:1]: KIV: Verbose retry mechanism from point of failure

File: utils.js
Comment (joshualai9922) [inline utils.js:1]: KIV: Verbose retry mechanism from point of failure for write, read, mkdir intermediate folders functions

**Pull request closed without merging**